### ISSUE TITLE
Support folders in bookmarks

### ISSUE THREAD

**Issue by lewadedun** [llelectronics/webcat#19]
Sometimes really miss it...

**Comment by llelectronics** [comment]
Yeah this is something that needs a little bit bigger change in the database to work but I guess I will put it on the todo list.
